packages/web/src/pages/saved-page/components/desktop/SavedPage.tsx

```diff
@@ -236,7 +236,6 @@ const SavedPage = ({
           playingIndex={playingIndex}
           scrollRef={mainContentRef}
           useLocalSort={allTracksFetched}
-          totalRowCount={dataSource.length}
```

Comment:
this was not the total row count, it was just the count how many tracks were on the current page